addons/payment/models/account_payment.py

```diff
@@ -15,6 +15,12 @@ class AccountPayment(models.Model):
             ('id', 'in', suitable_payment_token_ids),
         ]""",
         help="Note that only tokens from acquirers allowing to capture the amount are available.")
+    related_partner_ids = fields.Many2many(
+        comodel_name='res.partner', compute='_compute_related_partners')
```

Comment:
By the way, where is this field used? :thinking: (and I don't see the compute method either)

Reply:
Hm... Since it didn't ring any bells to me, I looked at the file, and it's used there for something I definitely didn't worked on. I think this was due to the rebase? Now the question is: what did I do to have github thinking I wrote that? oO